core/app/models/spree/wallet/add_after_order_complete.rb

```diff
@@ -0,0 +1,25 @@
+class Spree::Wallet::AddAfterOrderComplete
```

Comment:
Yes, I agree with @tvdeyen here. This will also open the possibility to add sources on other places. For example, adding bitcoin in "my account". Also.. naming things is hard :)